nawab_bot.py

```diff
@@ -150,9 +153,26 @@ def nawab_retweet_tweet(api):
 
 
 def main():
-   api = nawab_twitter_authenticate()
-   nawab_curate_list(api)
-   nawab_retweet_tweet(api)
+    
+    # default directory
+    def_dir = 'var/log'
+    if not os.path.exists(def_dir):
+        os.makedirs(def_dir)
+        os.chdir(def_dir)
+    # initialise the parser
+    parser = argparse.ArgumentParser(
+        description="Provide  path for logs destination"
```

Comment:
This is meant to be a global description for all the argparser. I don't think this is how it should be.